src/app/util/util.spec.ts

```diff
@@ -122,4 +123,26 @@ describe('Util', () => {
       });
   });
 
+  describe('.dateIsInPast()', () => {
```

Comment:
I suggest to add test to verify that date is in past is not allowed (a negative case). What do think?

Reply:
thank you. done